flutter_app/lib/widgets/display_all_logs_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:imacs/modules/get_airside_logs.dart';
+
+class LogReader extends StatelessWidget{
+
+  const LogReader ({super.key, required this.getAirsideLogs});
+
+  final GetAirsideLogs getAirsideLogs;
+   
+  @override
+  Widget build (BuildContext context){
+    return SizedBox(
+      height: 300,
+      child: AspectRatio(
+        aspectRatio: 16/9,
+        child: ListView.builder(
+          itemCount: getAirsideLogs.getFiles().length,
+          itemBuilder: (context, index){
+            return Padding (
+              padding: const EdgeInsets.all(8.0),
+              child: ElevatedButton (
```

Comment:
Maybe use a `ListTile` widget instead, this definitely works, but it's best practice to do it the other way.